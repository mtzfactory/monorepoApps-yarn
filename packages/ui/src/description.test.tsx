import { render, screen } from '@testing-library/react-native';

import Description from './description';

describe('Description', function () {
  it('Renders as expected', function () {
    const expectedText = 'Lorem ipsum';

    render(<Description>{expectedText}</Description>);

    const text = screen.getByText(expectedText);

    expect(text).toHaveTextContent(expectedText);

    expect(screen.toJSON()).toMatchSnapshot();
  });
});
