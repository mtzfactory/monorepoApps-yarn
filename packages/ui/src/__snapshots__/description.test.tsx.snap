// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description Renders as expected 1`] = `
<Text
  style={
    [
      undefined,
      {
        "color": undefined,
        "fontSize": undefined,
      },
    ]
  }
>
  Lorem ipsum
</Text>
`;
